packages/components/src/components/tabs/tabs.tsx

```diff
@@ -52,10 +76,11 @@ const Tabs = ({
         }
     }, []);
 
-    let initial_index_to_show, tab_width;
+    let initial_index_to_show = 0;
```

Comment:
why did you change it here? as I see, you set for active_index default value 0, so in the 83 line initial_index_to_show anyway will be at list 0. no needs to set it here, I guess..

Reply:
@yauheni-kryzhyk-deriv, as seen in the screenshot attached here, there is a scoping issue I think, the assignment inside the `useConstructor` hook is not recognised, so I have to initialise it regardless of what  i do inside the hook

<img width="1154" alt="before_assignment" src="https://user-images.githubusercontent.com/104334373/198211613-251efbaa-486a-4eb8-9f11-71112b5ffc69.png">

Reply:
Ok< understand it. So maybe it would be better decision to pass to useState initial value 0, and and left that variable unassigned as before?. Just  for avoiding some confusions while reading the code and seeing that we are assigning  0 to this variable two times :)
how do you think?

Reply:
Alright. I'll explore that and see